libs/base/core.cpp

```diff
@@ -650,6 +650,11 @@ int includes(String s, String searchString, int start) {
     return -1 != indexOf(s, searchString, start);
 }
 
+//%
```

Comment:
we should get rid of this C++ function and use typescript instead...

Reply:
Added to the typescript with Microsoft/pxt#5583